content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,237 @@
+# dvc.api.params_show()
+
+Load <abbr>parameters</abbr> (name and values) tracked in a <abbr>DVC
+project</abbr>.
+
+```py
+def params_show(
+    *targets: str,  # Optional
+    repo: Optional[str] = None,
+    stages: Optional[Union[str, Iterable[str]]] = None,
+    rev: Optional[str] = None,
+    deps: bool = False,
+) -> Dict:
+```
+
+## Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.params_show()
+```
+
+## Description
+
+Retrieves <abbr>params</abbr> keys and values from a <abbr>DVC project</abbr>
+and returns a JSON structure such as:
+
+```json
+{
+  "split": 0.2,
+  "seed": 20170428
+}
+```
+
+Without arguments, this function will retrieve all params from all tracked
+parameter files (used in any `dvc.yaml` file). This applies to the current
+project version when using Git (including any changes in the working tree).
+
+The function parameters (below) let you restrict what's retrieved.
+
+## Parameters
+
+- `*targets` - names of one or more valid parameter file to retrieve params
+  from. For example, `"params.py, myparams.toml"`. If no `targets` are provided,
+  all param files tracked in any `dvc.yaml` will be used by default. Note that
+  explicit targets don't have to be used in a `dvc.yaml`, however.
+
+- `repo` - specifies the location of the DVC project. It can be a URL or a file
+  system path. Both HTTP and SSH protocols are supported for online Git repos
+  (e.g. `[user@]server:project.git`). _Default_: The current project (found by
+  walking up from the current working directory tree).
+
+- `stages` - name or names of the stages to retrieve params from. If `None`, all
+  parameters from all stages will be retrieved. _Default_: `None`.
+
+- `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
+  a branch or tag name, a commit hash or an
+  [experiment](/doc/command-reference/exp) name). If `repo` is not a Git repo,
+  this option is ignored. _Default_: `None` (current working tree will be used)
```

Comment:
> _Default_: `None` (current working tree will be used)

Is this the same for all the other functions BTW? Currently they state that the default is `HEAD`.

Reply:
> Is this the same for all the other functions BTW? Currently they state that the default is HEAD.

Yes, it is the same for all the other functions, HEAD can be kind of misleading there

Reply:
Extracted to #3681